types/decentlab/DWS/uplink.js

```diff
@@ -145,6 +145,23 @@ function consume(event) {
   // Default values
   data.frequency = Math.round(sample.frequency * 100) / 100;
   data.weight = Math.round(sample.weight * 100) / 100 - tare;
+  data.weightKg = data.weight / 1000;
+
+  // Init state
+  if (event.state.lastWeighting === undefined) {
+    event.state.lastWeighting = data.weight;
+  }
+
+  // Calculate increment
+  data.incrementGram = data.weight - event.state.lastWeighting;
```

Comment:
should this number always be positive? otherwise I would call it difference

Reply:
yes, it should be positiv.